WordPress/src/main/res/layout/pages_item.xml

```diff
@@ -17,8 +17,37 @@
         style="@style/MySitePostItemTitle"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
+        app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
-        tools:text="@string/title" />
+        tools:text="This is a long title to check how the text behaves when its too long" />
 
+    <com.google.android.material.textview.MaterialTextView
+        android:id="@+id/status"
+        style="@style/MySiteCardPagesCardPagesItemStatus"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="4dp"
+        android:background="#F2F2F7"
+        tools:drawableStart="@drawable/ic_published_page_dashboard_card"
+        tools:text="@string/dashboard_card_page_item_status_published"
+        android:drawablePadding="@dimen/margin_medium"
+        app:layout_constraintStart_toStartOf="parent"
+        android:paddingStart="10dp"
```

Comment:
np (again, a bit early) - just a reminder to use `android:paddingHorizontal` and `android:paddingVertical`. :)

Reply:
Yup, Thanks for point it out. Done in [b4269b0](https://github.com/wordpress-mobile/WordPress-Android/pull/18337/commits/b4269b060abe8dc42ea4c0e36c0881d20840d884)